pkg/log/pipeline_reader.go

```diff
@@ -234,6 +238,13 @@ func hasPipelineRunFailed(prConditions duckv1beta1.Conditions) error {
 	return nil
 }
 
+func isPipelineRunRunning(prConditions duckv1beta1.Conditions) error {
+	if len(prConditions) != 0 && prConditions[0].Status == corev1.ConditionUnknown {
+		return fmt.Errorf("pipelinerun is still running: %s", prConditions[0].Message)
+	}
+	return nil
+}
+
```

Comment:
This should probably return a bool. It's not an error if the PipelineRun is still running. We should probably avoid printing to Stderr in this case.

Reply:
It will be like this `pipelinerun is still running: Tasks Completed: 0 (Failed: 0, Cancelled 0), Incomplete: 2, Skipped: 0`

Reply:
Thanks for the clarification. What you have makes sense to me then.

Reply:
Thanks, addressed